src/common/options/crimson.yaml.in

```diff
@@ -78,6 +78,18 @@ options:
   level: dev
   desc: default logical address space reservation for seastore objects' metadata
   default: 16777216
+- name: seastore_full_integrity_check
```

Comment:
Per discussion, as a longer term design, how about adding a TODO:
// TODO: implement sub-extent checksum and deprecate this configuration.